node/cli/src/chain_spec.rs

```diff
@@ -18,16 +18,18 @@
 
 use primitives::{AuthorityId, ed25519};
 use node_primitives::AccountId;
-use node_runtime::{GenesisConfig, ConsensusConfig, CouncilSeatsConfig, CouncilVotingConfig, DemocracyConfig,
-	SessionConfig, StakingConfig, TimestampConfig, BalancesConfig, TreasuryConfig, UpgradeKeyConfig,
-	ContractConfig, Permill, Perbill};
+use node_runtime::{ConsensusConfig, CouncilSeatsConfig, CouncilVotingConfig, DemocracyConfig,
+				   SessionConfig, StakingConfig, TimestampConfig, BalancesConfig, TreasuryConfig,
```

Comment:
The breaks our [house style](https://github.com/paritytech/polkadot/wiki/Style-Guide) which is enforced strictly.